microservices-connector/config/samples/chatQnA_v2.yaml

```diff
@@ -25,38 +25,30 @@ spec:
       - name: Embedding
         internalService:
           nameSpace: chatqa
-          serviceName: embedding-service
+          serviceName: embedding-svc
           config:
             no_proxy: ".chatqa.svc.cluster.local"
             http_proxy: "insert-your-http-proxy-here"
             https_proxy: "insert-your-https-proxy-here"
-            tei_endpoint: "http://tei-embedding-service.chatqa.svc.cluster.local:6006"
             endpoint: /v1/embeddings
       - name: TeiEmbedding
         internalService:
           nameSpace: chatqa
-          serviceName: tei-embedding-service
+          serviceName: tei-embedding-svc
           config:
             no_proxy: ".chatqa.svc.cluster.local"
             http_proxy: "insert-your-http-proxy-here"
```

Comment:
- we need remove all proxy setting in this sample
- rename this as chatQnA_xeon.yaml
- remove old chatQna.yaml
- remove old  gmc_v1alpha3_gmconnector.yaml

Reply:
done!